docs/UserGuide.md

```diff
@@ -80,27 +77,27 @@ applications.
 
 ### Adding an application : `add`
 
-Adds an application to a company to the list.
+Adds an application to the job application list.
 
-**Format:** `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+**Format:** `add c/COMPANY r/ROLE [d/DEADLINE] [s/STATUS] [i/INDUSTRY]`
 
 * Users must input a company `NAME`
-* Details of the company such as `PHONE_NUMBER`, `EMAIL` and `ADDRESS` are optional
+* Details of the company such as `DEADLINE`, `STATUS` and `INDUSTRY` are optional
 
 **Successful command:**
-print “Application to (Company name) has been added.”
+print “New job added: COMPANY; Role: ROLE; Status: STATUS; Deadline: DEADLINE; Industry: INDUSTRY"
 
 **Failed command:**
-print “Error: ” and error message for:
-
-* `NAME` is not in the command: “Name of company was not included in the command.”
+print "Invalid command format!
+add: Adds a job to the application book. 
+Parameters: c/COMPANY r/ROLE [Optional] d/DEADLINE s/STATUS i/INDUSTRY"
```

Comment:
Consider removing successful and failed command as it can get messy when we add more commands in the future (and a good error message should convey the error without it being in the UG)

Reply:
@JeremyYong128 @evanyan13 @migfoo02 thoughts? if yall ok, i will just remove the successful and failed commands